static/js/chart/types.ts

```diff
@@ -137,3 +141,20 @@ export interface MapPoint {
   latitude: number;
   longitude: number;
 }
+
+export interface RankingChartDataGroup {
```

Comment:
please add some documentation for the shared interface

Reply:
Done